src/libsync/configfile.h

```diff
@@ -45,7 +45,9 @@ class OWNCLOUDSYNC_EXPORT ConfigFile
 
     [[nodiscard]] QString configPath() const;
     [[nodiscard]] QString configFile() const;
+    [[nodiscard]] QString localConfigFile() const;
```

Comment:
It is impossible to understand what the difference between `configFile` and `localConfigFile` is from their names, and without really digging into the code. `localConfigFile` should have a name that is more indicative of the data it will hold